config/master.key

```diff
@@ -0,0 +1 @@
+bdcf59fcc4417e7e61bd6e2252d8babd
```

Comment:
- Guys you should never push your master.key file into source control. Pleas add it to `.gitignore` file. [Read more here](https://blog.capsens.eu/rails-5-2-imperfect-credentials-4ef03934aea4)